packages/terra-button-group/src/clinical-lowlight-theme/ButtonGroup.module.scss

```diff
@@ -13,8 +13,9 @@
 
     --terra-button-group-button-focus-background-color: #1a2023;
     --terra-button-group-button-focus-border-color: #181b1d;
-    --terra-button-group-button-focus-box-shadow: 0 0 1px 3px rgba(76, 178, 233, 0.5), 0 0 7px 4px rgba(76, 178, 233, 0.35);
-
+    --terra-button-group-button-focus-box-shadow: none;
+    --terra-button-group-button-focus-outline: 2px dashed #b2b5b6;
```

Comment:
this variable is no longer used in buttonGroup.module.scss

Reply:
As suggested below added outline and outline-offset  in mixins